editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -862,41 +872,78 @@ export function handleKeyDown(
         if (!isSelectMode(editor.mode)) {
           return []
         }
-        return [
-          EditorActions.applyCommandsAction(
-            editor.selectedViews.flatMap((elementPath) => {
-              const element = MetadataUtils.findElementByElementPath(
-                editor.jsxMetadata,
+
+        const commands = editor.selectedViews.flatMap((elementPath) => {
+          const element = MetadataUtils.findElementByElementPath(editor.jsxMetadata, elementPath)
+          if (element == null) {
+            return []
+          }
+
+          const contentAffectingType = getElementContentAffectingType(
+            editor.jsxMetadata,
+            editor.allElementProps,
+            elementPath,
+          )
+
+          if (contentAffectingType === 'fragment') {
+            return []
+          }
+
+          if (contentAffectingType === 'simple-div') {
+            const childrenBoundingFrame = getChildrenBoundingRectangle(
```

Comment:
You should just be able to use `MetadataUtils.getFrameInCanvasCoords` here instead. That would be safer too since it would take into account any text children

Reply:
🤦‍♂️ Sounds obvious in retrospect, thanks for pointing it out! https://github.com/concrete-utopia/utopia/pull/3382/commits/c37e140feccf4d03b6758940a5ec7140800c8892